xml/System.Runtime.InteropServices/ComWrappers+ComInterfaceEntry.xml

```diff
@@ -54,7 +54,7 @@
         <ReturnType>System.IntPtr</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
+        <summary>Memory must have the same lifetime as the memory returned from the call to <see cref="M:System.Runtime.InteropServices.ComWrappers.ComputeVtables(System.Object,System.Runtime.InteropServices.CreateComInterfaceFlags,System.Int32@)" />.</summary>
```

Comment:
This seems more like remarks than a summary. What can we put here instead?

Reply:
Yup, no problem.